yaml2ncml/yaml2ncml.py

```diff
@@ -92,12 +92,9 @@ def add_var_atts(text, a):
     else:
         if a['variables']['include']:
             vars = list(vars_all.intersection(vars_include))
-    rho_vars = [var for var, vart in ncv.items() if 'eta_rho' in
-                vart.dimensions and 'xi_rho' in vart.dimensions]
-    u_vars = [var for var, vart in ncv.items() if 'eta_u' in
-              vart.dimensions and 'xi_u' in vart.dimensions]
-    v_vars = [var for var, vart in ncv.items() if 'eta_v' in
-              vart.dimensions and 'xi_v' in vart.dimensions]
+    rho_vars = [var for var, vart in ncv.items() if 'eta_rho' in vart.dimensions and 'xi_rho' in vart.dimensions]
+    u_vars = [var for var, vart in ncv.items() if 'eta_u' in  vart.dimensions and 'xi_u' in vart.dimensions]
+    v_vars = [var for var, vart in ncv.items() if 'eta_v' in  vart.dimensions and 'xi_v' in vart.dimensions]
```

Comment:
@rsignell-usgs Should add what `pep8` rules you will not enforce.  That will avoid this kind of back-and-forth changes.